acceptance/tests/parser_functions/puppet_lookup_cmd.rb

```diff
@@ -2,6 +2,9 @@
 # doc:
 # https://docs.puppetlabs.com/puppet/latest/reference/lookup_quick_module.html
 
+host = agents.find { |host| host != master }
+skip_test "No agents present that are not running on a master" if host.nil?
+
```

Comment:
@joshcooper Yeah, thanks for pointing that out.  I obviously didn't really look at what this test is doing when I made this change.  I agree it doesn't really matter for the purpose of this test if there's an agent not running on the master - or even that there's an agent running anywhere for that matter.  I reworked my code change to just allow the two node names that this test is exercising for lookups be different from each other - using two hard-coded values which have nothing to do with the beaker host configuration.  This should hopefully allow this test to be run successfully if the configuration has just a single host using the 'master' and an 'agent' roles or any of the more common split node configurations that we typically run in CI.  I put up a replacement PR with this change as #5403 and am just going to close this one out.